pkg/report/linux.go

```diff
@@ -1673,7 +1673,7 @@ var linuxOopses = append([]*oops{
 			{
 				title: compile("WARNING: .*mm/vmalloc.c.*__vmalloc_node"),
 				fmt:   "WARNING: zero-size vmalloc in %[1]v",
-				stack: warningStackFmt("vmalloc"),
+				stack: warningStackFmt("v.?alloc"),
```

Comment:
We have vmalloc frame in other report formats as well and in skipPatterns. I think we need to skip it in all of them (it will show up sooner or later).